contrib/test/integration/main.yml

```diff
@@ -51,7 +51,7 @@
     - name: install Golang tools
       include: golang.yml
       vars:
-          version: "1.12.2"
+        version: "1.12.2"
```

Comment:
1.12.2 here and 1.12.6 above?  is that intentional?

Reply:
Hm, no I will fix it as soon as the build (hopefully) succeeds. Thanks :pray: